primary/src/block_waiter.rs

```diff
@@ -30,7 +30,7 @@ use types::{
 };
 use Result::*;
 
-const BATCH_RETRIEVE_TIMEOUT: Duration = Duration::from_secs(1);
+const BATCH_RETRIEVE_TIMEOUT: Duration = Duration::from_secs(10);
```

Comment:
Is this a temporary fix? If so should we add a TODO here to change it back with some tracking issue?

Reply:
Done!